package.json

```diff
@@ -20,7 +20,7 @@
     "format": "turbo format",
     "format:prettier": "prettier --write .",
     "format:syncpack": "syncpack format",
-    "lint": "turbo lint -- --max-warnings 0",
```

Comment:
why do want to make the linter less strict?

Reply:
if the linter uses --max-warnings 0, it's forced to fail on the first warning. this means you can't see all the warnings at once.

this PR is closed in favor of https://github.com/penumbra-zone/web/pull/1461 but, in general, i think if `--max-warnings 0` is applied anywhere, it should be in a ci workflow and not package.json.